src/arti/client.rs

```diff
@@ -0,0 +1,157 @@
+//! A general interface for Tor client usage.
```

Comment:
There is two `client.rs` in the sources. As I'm a bit new to the code base, what is the purpose of the "arti" subdir?

Reply:
The client is the `arti` directory is a low-level client much similar to the one in the `tor-client` crate in Arti. Maybe I can move this code into the higher-level client, but I will have to check that it does not require private functions from the custom dirmgr.